File: project-euler.java/src/main/java/euler/solutions/_6.java
package euler.solutions;

import euler.common.BaseSolution;

public class _6 extends BaseSolution {
    @Override
    public long solve() {
        int sum = 0, sumSqrs = 0;

        for (int i = 1; i <= 100; i++) {
            sum     += i;
            sumSqrs += i * i;
        }

        return sum * sum - sumSqrs;
    }
}

File: project-euler.go/solutions/_6/6.go
package _6

func Solve() int64 {
	var sum int64;
	var sumSqrs int64;

	for i := int64(1); i <= 100; i++ {
		sum     += i
		sumSqrs += i * i
	}

	return sum * sum - sumSqrs
}
